src/bundle-manifest.ts

```diff
@@ -150,6 +148,51 @@ export function generateBundles(depsIndex: TransitiveDependenciesMap):
 }
 
 /**
+ * Chains multiple bundle strategy functions together so the output of one
+ * becomes the input of the next and so-on.
+ */
+export function composeStrategies(strategies: BundleStrategy[]) {
+  return strategies.reduce((s1, s2) => (b) => s2(s1(b)));
+}
+
+/**
+ * Generates a strategy function which finds all non-entrypoint bundles which
+ * are dependencies of the given entrypoint and merges them into that
+ * entrypoint's bundle.
+ */
+export function generateEagerMergeStrategy(entrypoint: UrlString):
+    BundleStrategy {
+  return generateMatchMergeStrategy(
+      entrypoint,
+      b => b.entrypoints.has(entrypoint) && !getBundleEntrypoint(b));
```

Comment:
Commit 9ff1c27640d06adcb2f6998b08913103601e4100 took care of all instances of non-parenthesized anonymous function arguments.